### DISCUSSION TITLE
Simple Time field for duration

### DISCUSSION THREAD

**Discussion by doucwiki** [Budibase/budibase#9435] [Help]
Hi!
Maybe some kind of newbie question since I started with budibase recently but couldn’t find  anything after days of searching. 
I’m looking for a solution for a simple time field to enter a duration. For example a duration of a task. I managed to calculate time differences from my datetime mysql databases via JS but I need to enter a duration in the format of hh:mm to pass back to my SQL database. 

Maybe someone can push me in the right direction. Thanks. 

PS:
Current Budibase release self hosted on docker,
Connected to MySQL Database

**Comment by melohagan** [comment]
Hey @doucwiki 

You could possibly have a look at: https://github.com/melohagan/budibase-component-kasper-date-picker

It's a [custom plugin](https://docs.budibase.com/docs/custom-plugin) that allows you to specify a custom date/time format.

**Reply by doucwiki** [reply]
Great plugin. 
Thanks. Tried it and it gets me a little closer to what I want.

**Reply by melohagan** [reply]
If you find any issues, want any additions, then feel free to raise issues against that repo!
